komodo/symlink/create_links.py

```diff
@@ -17,6 +17,13 @@ def working_dir(path):
         os.chdir(prev_dir)
 
 
+def traverse_symlink_tree(key, link_dict):
+    values = [k for k, v in link_dict.items() if v == key]
+    for value in values:
+        traverse_symlink_tree(value, link_dict)
+        print(f"{value} was implicitly moved as a side effect")
```

Comment:
I think we can drop "as a side effect". This is the main effect that we target after all.

Reply:
Yes. sounds good!